doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -75,7 +90,8 @@ This type of firmware upgrade supports updating the modem firmware using the ser
 The serialized firmware file uses the :file:`.cbor` extension.
 
 This DFU target downloads the serialized modem firmware to an external flash memory.
-Once the modem firmware has been downloaded, the library uses :ref:`lib_fmfu_fdev` to write the firmware to the modem.
+Once the modem firmware has been downloaded, the application should use :ref:`lib_fmfu_fdev` to write the firmware to the modem.
```

Comment:
should or must?

Reply:
Or may.. or can..

I tried to clarify that this library does not do the update, it is the `lib_fmfu_fdev` which does it. So application can use that.